### PR TITLE
HV-158 Move willChange Hack

### PR DESCRIPTION
**Pull request by timmccall-wf**
Problem
-------

The `willChange` CSS hack introduced an automatic minimum duration for zooming the AwesomeMap, which is undesirable.

Solution
--------

Move the timing of the hack such that it is triggered whenever the scale is reported as changed to the AwesomeMap.

How To +10/QA
-------------

@robbecker-wf

### CODE REVIEW

**Comment by rmconsole-wf** [comment]
When this pull is merged I will not be able to function
		properly because no ticket is included in the title.

***
*Last updated on Tuesday, January 27 11:57 PM CST*

File: src/scroll_list/ScrollList.js [hunk 1]
```diff
@@ -1245,10 +1245,7 @@ define(function(require) {
             options = options || {};
             (this.getCurrentItemMap() || this._listMap).zoomTo({
                 scale: scale,
-                // HACK: always give at least 10 ms duration so that
-                // TransformUtil.animate gets used instead of the
-                // simulated zoom
-                duration: Math.max(10, options.duration || 0),
+                duration: options.duration,
```
Comment (robbecker-wf) [inline src/scroll_list/ScrollList.js]: that feels good

**Comment by robbecker-wf** [comment]
+1 Have you profiled paints to make sure it's not over painting with this change? screenshot?

**Comment by todbachman-wf** [comment]
+10. Tested in unison with HV-129.

**Comment by charliekump-wf** [comment]
QA Resource Approval: +1

- [x] Testing instruction
- [x] Dev +1's
- [x] Dev/QA +10 with detail of what was tested
- [ ] Unit test created/updated  
    - Not necessary.
- [x] All unit tests pass 

Merging into master.

**Comment by charliekump-wf** [comment]
@rmconsole-wf

**Pull request merged**